.vscode/settings.json

```diff
@@ -22,10 +22,7 @@
     "Cargo.toml"
   ],
   "rust-analyzer.cargo.targetDir": true, // Don't let RA blocking cargo build
-  "rust-analyzer.cargo.features": [
```

Comment:
found RA broken when develop deny-ext feature caused by clippy may pass non-supported features to crate